README.md

```diff
@@ -80,24 +79,181 @@ After you input your application credentials, you can click the setup button, wh
 
 **Important**: Make sure your Resources/MobiledgeXSettings.asset file has the correct information for your application. 
 
-Once that setup has been completed, you can very easily call all the necessary API requests to connect to a cloudlet with your application deployed. Here is some example code using the MobiledgeXIntegration class that comes with the package : 
+Once that setup has been completed, you can very easily call all the necessary API requests to connect to a cloudlet with your application deployed. Here is some example code using the MobiledgeXIntegration class that comes with the package 
+
+
+**Getting Edge Connection Url**
+
+
 
 ```csharp
 using MobiledgeX;
+using DistributedMatchEngine;
+using UnityEngine;
+using System.Collections;
+
+[RequireComponent(typeof(MobiledgeX.LocationService))]
+public class NetworkManager : MonoBehaviour
```

Comment:
replace NetworkManager with YourClassName. otherwise a developer might run into a namespace conflict since NetworkManager is a very commonly used name